swagger.yaml

```diff
@@ -6853,7 +6853,7 @@ definitions:
       - 'ingress-vips-defined'
       - 'ingress-vips-valid'
       - 'all-hosts-are-ready-to-install'
-      - 'sufficient-masters-count'
+      - 'sufficient-masters-and-workers-count'
```

Comment:
I'm not sure it's a good idea to change it - it can break the UI and also partners that are using the API
@avishayt WDYT?

Reply:
This validations are only exposed to the user via `validations` field in the cluster object. The UI pops a message if one of the cluster validations is failing, regardless of its name. Therefore, it shouldn't break the UI (not breaking from my tests as well). It could break QE / UI tests, but this could be addressed in advance. I am not sure we want to keep using this name as it is confusing. We can probably keep the old one as well just for back compatibility

Reply:
the UI has a mapping between this value and the string that is being displayed to the user.
I think it's better to keep the old one and add a new one - I also recommended to change it to sufficient-hosts-count because we will have more host roles in the near future.

Reply:
Ack. I will put the old name back